karma.conf.js

```diff
@@ -0,0 +1,62 @@
+// Karma configuration
+// Generated on Tue Dec 08 2015 23:01:01 GMT-0800 (Pacific Standard Time)
+
+module.exports = function (config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['browserify', 'jasmine'],
```

Comment:
to resolve commonjs lookup, using `karma--browserify` to bundling modules. there is `karma-commonjs` does job without bundling, but has some issues of resolving node_modules (https://github.com/karma-runner/karma-commonjs/issues/29) blocks adaptation.